Inc/eeprom.h

```diff
@@ -11,7 +11,15 @@ typedef union EEprom_u {
             uint8_t major; //3
             uint8_t minor; //4
         } version;
-        char firmware_name[12]; //5-16
+        uint8_t max_ramp;    // 0.1% per ms to 25% per ms 
```

Comment:
it is nice to have the offset of each variable like we do in the rest of the structure